languages/states.php

```diff
@@ -853,22 +851,22 @@
 	),
 	'CK' => array(),
 	'CL' => array(
-		'AN' => __( 'Antofagasta Region', 'lifterlms' ),
-		'AR' => __( 'Araucanía Region', 'lifterlms' ),
-		'AP' => __( 'Arica y Parinacota Region', 'lifterlms' ),
-		'AT' => __( 'Atacama Region', 'lifterlms' ),
-		'AI' => __( 'Aysén Region', 'lifterlms' ),
-		'BI' => __( 'Bío Bío Region', 'lifterlms' ),
-		'CO' => __( 'Coquimbo Region', 'lifterlms' ),
-		'LL' => __( 'Los Lagos Region', 'lifterlms' ),
-		'LR' => __( 'Los Ríos Region', 'lifterlms' ),
-		'MA' => __( 'Magellan and the Chilean Antarctic Region', 'lifterlms' ),
-		'ML' => __( 'Maule Region', 'lifterlms' ),
-		'LI' => __( 'O\'Higgins', 'lifterlms' ),
-		'RM' => __( 'Santiago Metropolitan Region', 'lifterlms' ),
-		'TA' => __( 'Tarapacá Region', 'lifterlms' ),
+		'AI' => __( 'Aisén del General Carlos Ibañez del Campo', 'lifterlms' ),
```

Comment:
Cannot find any occurrence of "Aisén", "Aysén" seems the correct one.
What's the source of all these changes?

Reply:
https://github.com/gocodebox/lifterlms/tree/trunk/languages#localization-information-files

Reply:
Looks like this is the source of the updates to Chile: https://github.com/dr5hn/countries-states-cities-database/issues/255

Reply:
On wikipedia I see that "Aysén" is the official spelling and it is sometimes spelled with an i instead of a y:

The first line has this: https://en.wikipedia.org/wiki/Ays%C3%A9n_Region

Reply:
Alright, I think nobody wiill complain...